pkg/sif/update.go

```diff
@@ -33,15 +33,15 @@ func OptUpdateTempDir(d string) UpdateOpt {
 	}
 }
 
-// Update modifies the SIF file associated with fi so that it holds the content
+// Update modifies the SIF file associated with f so that it holds the content
 // of ImageIndex ii. Any blobs in the SIF that are not referenced in ii are
 // removed from the SIF. Any blobs that are referenced in ii but not present in
 // the SIF are added to the SIF. The RootIndex of the SIF is replaced with ii.
 //
 // Update may create one or more temporary files during the update process. By
 // default, the directory returned by os.TempDir is used. To override this,
 // consider using OptUpdateTmpDir.
-func Update(fi *sif.FileImage, ii v1.ImageIndex, opts ...UpdateOpt) error {
+func (f *OCIFileImage) Update(ii v1.ImageIndex, opts ...UpdateOpt) error {
```

Comment:
Would `UpdateRootIndex` (or similar) make the functionality clearer here?

Reply:
Agreed.